### DISCUSSION TITLE
How to stop quarkus keycloak

### DISCUSSION THREAD

**Discussion by PavanKalyan-Gandhapudi** [keycloak/keycloak#20346] [Q&A] [answered]
Like we have kc.bat start and kc.bat restart, do we have anyway to stop keycloak. Apart from ctrl+c, is there any other way? I have to terminate/stop the keycloak through the code.
I tried graceful shutdown by providing timeout. But its not working. 
quarkus.shutdown.timeout=60.

**Comment by sschu** [comment, marked as answer]
Have you tried `taskkill`https://learn.microsoft.com/en-us/windows-server/administration/windows-commands/taskkill ?
